dvm/dvm_result.go

```diff
@@ -90,6 +95,7 @@ func (this DVMResult) MarshalJSON() ([]byte, error) {
 		ContractMethodExecResult string `json:"contractMethodExecResult"`
 		Divvy                    int64  `json:"divvy"`
 		HertzCost                uint64 `json:"hertzCost"`
+		Logs                     string `json:"logs"`
```

Comment:
```suggestion
		Logs                     string `json:"logs,omitempty"`
```

Reply:
After more review.  This is in the DVM result object which will only be populated when you deploy and execute a contract.  The logs value should never be empty, so I don't think it is necessary to have the omitempty.